File: chrome/test/data/extensions/autoplay_iframe/frame.js
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2018 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

window.addEventListener('message', function() {
  var audio = document.createElement('audio');
  audio.src = 'test.mp4';
  audio.play().then(() => {
    top.postMessage('autoplayed', '*');
  }, e => {
    top.postMessage(e.name, '*');
  });
});

File: base/profiler/frame.cc
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2019 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "base/profiler/frame.h"

namespace base {

Frame::Frame(uintptr_t instruction_pointer, const ModuleCache::Module* module)
    : instruction_pointer(instruction_pointer), module(module) {}

Frame::~Frame() = default;

}  // namespace base
